build/pmm-server-docker/Dockerfile

```diff
@@ -5,15 +5,16 @@ EXPOSE 80 443
 WORKDIR /opt
 
 RUN useradd -s /bin/false pmm
-RUN yum -y install epel-release && yum -y install ansible
+RUN yum -y install epel-release \
+    && yum -y install ansible
```

Comment:
There is not real need to use `&&` – we squash all image layers

Reply:
This change will be reverted. It was trying to fix installation issue by adding priorities plugin for yum.